spring-cloud-azure-config/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>spring-cloud-azure</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-cloud-azure-config</artifactId>
+    <name>Spring Cloud Azure Config Service Configuration</name>
```

Comment:
`Spring Cloud Azure Config` should be enough. See Consul's name at [here](https://github.com/spring-cloud/spring-cloud-consul/blob/master/spring-cloud-consul-config/pom.xml#L9).
We can explain more in `description` tag though.